CIP-XXXX/README.md

```diff
@@ -0,0 +1,377 @@
+
+1. Fairly selecting which stake pool(s) would receive delegation presents
+   substantial administrative burden on the budget administrator.
+2. Pool(s) receiving delegation may be harmed by approaching or becoming
+   saturated from this ephemeral delegation, prohibiting the growth of
+   long-term, sustainable delegation.
+
+Using custodial funds in governance is similarly problematic:
+
+1. Given on-chain governance actions may include the approval of future budgets
+   or spending that would directly benefit the administrators, there is a clear
+   conflict of interest in using custodial Ada for on-chain governance.
+2. The custodial Ada held by the administrator is on behalf of the entire
+   Cardano community. It would be impossible to fairly and accurately represent
+   the will of the entire community through any on- or off-chain governance
+   systems.
+
+<hr />
+
+#### Rule: Administrator Transparency
+
+##### Definition
+
+Budget administrators and custodians **MUST** be non-anonymous, legally
+recognized, public entities with a publicly disclosed board of directors and/or
+responsible parties.
+
+##### Rationale
+
+Individual budgets are likely to account for large amounts of value both in Ada
+and fiat computation. The responsible parties to manage and oversee the budget
+process should be publicly disclosed to bring confidence and accountability to
+the Cardano community.
+
+<hr />
+
+#### Rule: Radical Transparency
+
+##### Definition
+
+Budget allotments earmarked for specific purposes (buckets) **MUST** be kept
+separate and **MUST** be kept in secure, publicly disclosed addresses until
+disbursed as part of the normal budget process.
+
+##### Rationale
+
+If a single custodian is responsible for multiple facets of the budget (core
+development and research, for example) then it is imperative that the community
+be capable of auditing the funds distributed from the Treasury to the custodian
+and subsequently to contractor(s) engaging in work as part of the budget
+process. Co-mingling (pooling) funds from various buckets decreases transparency
+and should not be tolerated.
+
+<hr />
+
+#### Rule: Conflict Resolution
+
+##### Definition
+
+A budget proposal **MUST** define a jurisdiction and conflict resolution clause.
+Custodial funds **MUST NOT** be used for court or legal expenses by the
+administrator without an explicit _governance action_ authorization from the
+Cardano Community following the same criteria as budget approval.
+
+##### Rationale
+
+By defining a _forum selection clause_ within the scope of the
+community-approved budget we can reduce uncertainty for both the administrators
+and the community should a conflict or dispute arise.
+
+<hr />
+
+#### Rule: Administrator Fees
+
+##### Definition
+
+A budget administrator and custodian, in total, **MUST NOT** charge more than 5%
+of the total budget amount for administrative costs. All administrative costs
+**MUST** be defined as part of the budget proposal.
```

Comment:
Perhaps the Civic Committee can draft a Guideline on Compensation for Committee Members, since this will form the bulk of Administrator Fees

Reply:
This naively (IMO) assumes that Intersect will be the only budget proposer and/or administrator. Other future administrators of budget may not have committees and other things but may still charge some amount of administration fees.

Reply:
agreed, this is independent issue but the bulk of administrators will likely be from Intersect or another MBO, which should still have a guideline on compensations for transparency and to filter out those who try to find ways to embezzle treasury.